lib/pf/radius.pm

```diff
@@ -169,7 +170,6 @@ sub authorize {
         ifIndex => $port,
         ifDesc => $ifDesc,
         user_name => $user_name,
-        username => $user_name,
```

Comment:
We have to keep it since we use it in the radius filter.

Reply:
my bad, this shouldn't have been part of the PR